batteryParameter.js

```diff
@@ -0,0 +1,90 @@
+class SOC {
+	static name = 'State of Charge';
```

Comment:
Variables like  name, min, max, thresholdpercentage, warningthreshold, enable warning seems like attributes of an entity. Code can be organised in a better way 👍